### DISCUSSION TITLE
Deleting an item in the Territory view SHOULD NOT also delete the item in the Observation view.

### DISCUSSION THREAD

**Discussion by okdistribute** [digidem/mapeo-desktop#560] [Ideas]
Deleting an item in the Territory view SHOULD NOT also delete the item in the Observation view. 

 

> gregor: Hi @karissa, great work on this. I agree with @aliya-ryan  on this - I think observations should not be deleted when their corresponding point is deleted from the map view.

> 06:17 <gregor> I think for this first pass at this for WCMC it is ok that deleting from the territory view deletes from both, but we should work towards a solution that does not do this.

> 06:19 <gregor> The reason for this is because observations act as an (important) permanent record of points taken in the field. I think generally we should be teaching that they are not deleted except when (a) it is training data or (b) there is duplicate data.

> 06:21 <gregor> When working with map data, there could be observations that do not make sense to be on the map, and the user would want to delete them. For example, if a team had gone out to document an oil spill, they might have taken 2-3 observations of the spill. The user might want to keep a single point "oil spill" on the map, and remove the other points, but the original observations (with their photos) remain an important record of that oil spill. I have
06:21 some further ideas about this I want to write up once I get them clear in my head.

**Comment by okdistribute** [comment]
cc @gmaclennan for more context

**Comment by okdistribute** [comment]
> @noffle : if it was HIDE FROM THIS VIEW instead of DELETE, it'd be easier to implement
12:51 then it could just a tag
12:51 and the client filters them"

**Comment by hackergrrl** [comment]
> When working with map data, there could be observations that do not make sense to be on the map, and the user would want to delete them. For example, if a team had gone out to document an oil spill, they might have taken 2-3 observations of the spill. The user might want to keep a single point "oil spill" on the map, and remove the other points, but the original observations (with their photos) remain an important record of that oil spill.

It would be good to try and confirm some of these assumptions with Mapeo users. This change could be quite a big implementation job, so being sure this is wanted & needed feels important.

**Comment by okdistribute** [comment]
If it's just 'hide from this view' we can probably implement this in github.com/digidem/osm-p2p-server using a filter
